### DISCUSSION TITLE
api rule multiple condition

### DISCUSSION THREAD

**Discussion by apujayed** [pocketbase/pocketbase#3586] [Q&A]
can we use multiple condition in api rule?

like when  
```
req.auth.Account_type="bidder"
@request.data.price_max >= price_max
```

and
```
req.auth.Account_type="broker"
@request.data.price_max >= price_max
```

**Comment by ganigeorgiev** [comment]
To combine conditions you can use parenthesis `(...)` and the `||` (_OR_) and `&&` (_AND_) operators:

```js
(req.auth.Account_type="bidder" || req.auth.Account_type="broker") &&
@request.data.price_max >= price_max
```

_More details and some examples you can find in https://pocketbase.io/docs/api-rules-and-filters/#filters-syntax._

**Reply by apujayed** [reply]
req.auth.Account_type="broker"
@request.data.price_max = price_max

little bid change here if ac type broker than this rule apply

**Reply by apujayed** [reply]
`(@request.auth.reference.Account_type = 'Broker')  && @request.data.price_max >= price_max (@request.auth.reference.account_type = 'Bidder') && @request.data.price_max > price_max` this api rule  not work
